setup.py

```diff
@@ -445,7 +445,8 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     "_libs.algos": {
         "pyxfile": "_libs/algos",
         "include": klib_include,
-        "depends": _pxi_dep["algos"],
+        "depends": _pxi_dep["algos"] + tseries_depends,
+        "sources": ["pandas/_libs/tslibs/src/datetime/np_datetime.c"],
```

Comment:
So the point of the capsule is you don't specify `sources` any more. Since you are adding a new function here, you'll want to add it to the datetime capsule itself

Reply:
I’m still struggling here with builds failures I can reproduce. Any other thoughts?